ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -900,6 +900,19 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
          prov->domain_attr->mr_mode = FI_MR_BASIC;
     }
 
+#if OPAL_CUDA_SUPPORT
+    /**
+     * Some providers do not require the use of the CUDA convertor
+     * in OMPI and its use will cause performance degradation. The
+     * following providers will disable it when selected.
+     */
+    if (!strncmp(prov->fabric_attr->prov_name, "psm3", 4)
+        || !strncmp(prov->fabric_attr->prov_name, "psm2", 4))
```

Comment:
@mwheinz, Since psm2 is now maintained by Cornelis, just wanted your approval to add psm2 here before it is merged. Most of my testing was on psm2 as we prep for full CUDA support in psm3, but I can remove it if you want to do your own testing here?

Reply:
I gave it a quick smoke test and it looked fine to me. Haven’t had a chance to duplicate your performance tests, but I know your test environment so I’m comfortable with it.